packages/renderer/src/lib/image/NoContainerEngineEmptyScreen.svelte

```diff
@@ -1,16 +1,8 @@
 <script lang="ts">
 import Fa from 'svelte-fa/src/fa.svelte';
-import { faPaste } from '@fortawesome/free-solid-svg-icons';
-import { onMount } from 'svelte';
+import EmptyScreen from '../ui/EmptyScreen.svelte';
 </script>
 
-<div class="h-full min-w-full flex flex-col">
-  <div class="pf-c-empty-state h-full">
-    <div class="pf-c-empty-state__content">
-      <i class="fas fa-layer-group pf-c-empty-state__icon" aria-hidden="true"></i>
-
-      <h1 class="pf-c-title pf-m-lg">No Container Engine</h1>
-      <div class="pf-c-empty-state__body">Configure/Start a container engine first</div>
-    </div>
-  </div>
-</div>
+<EmptyScreen title="No Container Engine" message="Configure/Start a container engine first">
+  <span slot="icon"><i class="fas fa-layer-group pf-c-empty-state__icon" aria-hidden="true"></i></span>
```

Comment:
Probably we should directly provide the icon class name instead of providing a slot html object

```html
<EmptyScreen iconClass="fas fa-layer-group">
```

or reuse fa objects
```typescript
import { faLayerGroup } from '@fortawesome/free-solid-svg-icons';
```
```html
<EmptyScreen icon="{faLayerGroup}" title="No Container Engine" message="Configure/Start a container engine first">
```

Reply:
The reason for slot vs hard-coding fa was that today we have different icons in nav (svg) vs lists and empty screens (fa). I'm assuming we'll want to make these consistent at some point and either svgs will win or we'll have a mix.

Reply:
we could provide name of the svelte component for SVG part and the fontAwesome component

```
iconSvg="ContainerIcon" iconFas="{faLayerGroup}"
```

or name of the svg or font awesome
```html
icon="svg:ContainerIcon" or icon="fa:faLayerGroup"
```

Reply:
also it should be possible to do
```
  icon="{faLayerGroup}" or icon="{ContainerIcon}"
```

Reply:
Thanks, agreed this is much nicer. Fixed in new commit.